src/assets/style.scss

```diff
@@ -140,103 +260,92 @@ table {
-  float: right;
-  clear: right;
-  width: 35%;
-  background: $bg-overlay;
-  border: 1px solid $border-color;
-  border-bottom-width: 0;
-  border-radius: $border-radius;
-  margin: 0 0 $padding-containers $padding-containers;
-
-  figcaption {
-    font-style: italic;
-  }
-  section, figcaption {
-    display: flow-root;
-    border-bottom: 1px solid $border-color;
-    padding: 0 $padding-containers;
-  }
-  section.header {
-    text-align: center;
-  }
-  section.img, img {
-    padding: 0;
-  }
-}
-
-.content {
-  padding: 0 $padding-containers $padding-containers $padding-containers;
+.content-article {
+  max-width: $width-content;
+  margin-right: auto;
   display: flow-root;
+  padding: 0 $padding-containers;
   background: $bg-content;
   border: 1px solid $border-color;
   border-radius: $border-radius $border-radius 0 0;
 
-  h1 {
-    border-bottom: 1px solid $border-color;
-    padding-bottom: 10px;
-    overflow: hidden;
-  }
-
-  p {
-    line-height: 1.5;
-  }
-
-  strong {
-    color: $fg-strong;
-  }
-
-  blockquote {
-    display: flow-root;
-    font-style: italic;
-    background: $bg-inset;
+  .metabox, figure, aside {
+    float: right;
+    clear: right;
+    width: 35%;
+    background: $bg-overlay;
+    border: 1px solid $border-color;
+    border-bottom-width: 0;
     border-radius: $border-radius;
-    padding: 0 $padding-containers;
-    margin-left: 0;
-  }
+    margin: 0 0 $padding-containers $padding-containers;
 
-  pre, code {
-    font-family: $font-family-code;
-    font-size: $font-size-code;
-    background: $bg-inset;
-    border-radius: $border-radius;
-  }
-
-  code {
-    padding: 0.2em;
-  }
-
-  pre {
-    padding: 1em;
-    overflow-x: auto;
-    code {
+    figcaption {
+      font-style: italic;
+    }
+    section, figcaption {
+      display: flow-root;
+      border-bottom: 1px solid $border-color;
+      padding: 0 $padding-containers;
+    }
+    section.header {
+      text-align: center;
+    }
+    section.img, img {
       padding: 0;
-      border: none;
-      background: none;
     }
   }
 }
 
 .content-footer {
+  grid-area: footer;
   margin-bottom: $padding-containers;
   padding: 0 $padding-containers;
   display: flow-root;
-  background: $bg-overlay;
-  border: 1px solid $border-color;
-  border-radius: 0 0 $border-radius $border-radius;
-  text-align: right;
+  text-align: center;
+  max-width: $width-content;
   border-top: 0;
   color: $fg-faded;
 
   a {
     color: $fg-faded;
   }
 }
+
+.content-layout {
+  display: grid;
+  grid-template-rows: max-content 1fr;
+  grid-template-columns: 10% 80%;
+  grid-template-areas:
+    "sidebar main"
+    "sidebar footer";
+}
+
+//mobile view
+@media all and (max-width: 40em) {
+  .content-layout {
+    grid-template-columns: 100%;
+    grid-template-areas: "main" "sidebar" "footer";
+  }
+
+  .top-header {
+    .c20-name-short {
+      display: inline-block;
+    }
+    .c20-name-long {
+      display: none;
+    }
+  }
+}
```

Comment:
Mobile-size styles. We can change the layout of the page and hide some unnecessary elements.